clumper/clump.py

```diff
@@ -950,6 +950,28 @@ def head(self, n=5):
         n = min(n, len(self))
         return self._create_new([self.blob[i] for i in range(n)])
 
+    def sample(self, n: int, replace: bool, weights, random_state: int):
+        """Samples n data from the collection
+
+        Arguments:
+            n (int): the number of items to sample
```

Comment:
If the types are given in the method there's no need to add them in the docstring. The doctool will handle this automatically.

Reply:
These were automatically added by my docstring generator so if it's not a problem, I would like to just keep it.

Reply:
I'd prefer to remove them if the docs rendered strangely because of it. We're not using standard docstrings in this package in favor of a more modern markdown template. 

When you run `mkdocs serve` locally, what does this render as?

Reply:
![image](https://user-images.githubusercontent.com/47146983/110701496-8269b800-81f1-11eb-81c7-91318798d55b.png)

It seems to work fine.

Reply:
@koaning  any comments?